test/powershell/Language/Scripting/NativeExecution/NativeStreams.Tests.ps1

```diff
@@ -12,9 +12,9 @@ Describe "Native streams behavior with PowerShell" -Tags 'CI' {
         $error.Clear()
 
         $command = [string]::Join('', @(
-            '[Console]::Error.Write(\"foo`n`nbar`n`nbaz\"); ',
-            '[Console]::Error.Write(\"middle\"); ',
-            '[Console]::Error.Write(\"foo`n`nbar`n`nbaz\")'
+            '[Console]::Error.Write("foo`n`nbar`n`nbaz"); ',
```

Comment:
Is the backslash escape not needed or will it now break existing code that used this pattern?

Reply:
99e8f57 should revert this behavior. I just forgot to change it.

Reply:
It will break such over-esacpes.
Hence the experimental switch.